numerics_paper/version2/numerical_chj.tex

```diff
@@ -689,15 +667,16 @@ \subsection{Dam break problem on a dry bed}\label{sec:rp_dry_bed}
 at transonic rarefactions. Since the baseline Roe solver we use (see \S\ref{sec:roe}) does not contain
 an entropy fix, it is important to demonstrate that the blended Riemann solver fixes the entropy glitch.
 We follow the setup in \cite[\S 4.1.2]{delestre2013swashes}.
-The domain is given by $\Omega=(0,L=10)$, and the initial condition is $hu(x,0)=0$ and
+The domain is given by $x\in(0,10)$, and the initial condition is $hu(x,0)=0$ and
 \begin{align*}
   h(x,0)=
   \begin{cases}
-    h_l, & \mbox{ if }, x\leq x_0=5 \\
-    h_r, &\mbox{ otherwise},
+    h_l & x\leq x_0=5 \\
+    0 & x >5,
   \end{cases}
 \end{align*}
-with $h_l=5\times 10^{-3}$ and $h_r=0$. In our experiments we use $h_r=1\times 10^{-15}$ to avoid division by zero.
+with $x_0=5$ and $h_l=5\times 10^{-3}$.  In our experiments we use $h_r=1\times
```

Comment:
x_0=5 is defined in the equation, maybe we don't need to include it here again. You erased h_r from the equation but left it here. I think it is better to include it in the equation so that we can say the value we use in the code.

Reply:
Fixed.